test/e2e/acl_logging.go

```diff
@@ -163,6 +163,328 @@ var _ = Describe("ACL Logging for NetworkPolicy", func() {
+		nsNames[2] = "anp-peer-open"
+		nsNames[3] = "anp-peer-unknown"
+		for _, ns := range nsNames[1:] {
+			_, err = e2ekubectl.RunKubectl("default", "create", "ns", ns)
+			Expect(err).NotTo(HaveOccurred())
+		}
+
+		By("creating pods in subject and peer namespaces")
+		cmd := []string{"/bin/bash", "-c", "/agnhost netexec --http-port 8000"}
+		for _, ns := range nsNames {
+			pod := newAgnhostPod(ns, fmt.Sprintf("pod-%s", ns), cmd...)
+			pod = e2epod.PodClientNS(fr, ns).CreateSync(context.TODO(), pod)
+			Expect(waitForACLLoggingPod(fr, ns, pod.GetName())).To(Succeed())
+			pods = append(pods, *pod)
+			framework.Logf("Created %s in namespace %s", pod.Name, pod.Namespace)
+		}
+	})
+	AfterEach(func() {
+		By("deleting the admin network policy")
+		_, err := e2ekubectl.RunKubectl("default", "delete", "anp", anpName, "--ignore-not-found=true")
+		Expect(err).NotTo(HaveOccurred())
+		By("deleting the baseline admin network policy")
+		_, err = e2ekubectl.RunKubectl("default", "delete", "banp", "default", "--ignore-not-found=true")
+		Expect(err).NotTo(HaveOccurred())
+		By("deleting subject and peer namespaces that are selected by the admin network policy")
+		for _, ns := range nsNames {
+			_, err := e2ekubectl.RunKubectl("default", "delete", "ns", ns, "--ignore-not-found=true")
+			Expect(err).NotTo(HaveOccurred())
+		}
+	})
+
+	It("the ANP ACL logs have the expected log level", func() {
+
+		By("sending traffic between acl-logging test pods we trigger ALLOW ACL logging")
+		clientPod := pods[0] // subject pod
+		pokedPod := pods[1]  // peer pod
+		framework.Logf(
+			"Poke pod %s (on node %s) from pod %s (on node %s)",
+			pokedPod.GetName(),
+			pokedPod.Spec.NodeName,
+			clientPod.GetName(),
+			clientPod.Spec.NodeName)
+		Expect(
+			pokePod(fr, clientPod.GetName(), pokedPod.Status.PodIP)).NotTo(HaveOccurred(),
+			"traffic should be allowed since we use an ALLOW all traffic policy rule")
+
+		By("verify the ALLOW ACL log level at Tier1")
+		clientPodScheduledPodName := pods[0].Spec.NodeName
+		// Retry here in the case where OVN acls have not been programmed yet
+		composedPolicyNameRegex := fmt.Sprintf("ANP:%s:Egress:0", anpName)
+		Eventually(func() (bool, error) {
+			return assertACLLogs(
+				clientPodScheduledPodName,
+				composedPolicyNameRegex,
+				allowACLVerdict,
+				initialAllowACLSeverity)
+		}, maxPokeRetries*pokeInterval, pokeInterval).Should(BeTrue())
+
+		By("sending traffic between acl-logging test pods we trigger DENY ACL logging")
+		clientPod = pods[0] // subject pod
+		pokedPod = pods[2]  // peer pod
+		framework.Logf(
+			"Poke pod %s (on node %s) from pod %s (on node %s)",
+			pokedPod.GetName(),
+			pokedPod.Spec.NodeName,
+			clientPod.GetName(),
+			clientPod.Spec.NodeName)
+		Expect(
+			pokePod(fr, clientPod.GetName(), pokedPod.Status.PodIP)).To(HaveOccurred(),
+			"traffic should be denied since we use an DENY all traffic policy rule")
+
+		By("verify the DENY ACL log level at Tier1")
```

Comment:
does tier 1 mean ANP, instead of BANP?

Reply:
yup tier1 = ANP ACLs evaluated 1st
tier2 = networkpolicy, egressfirewall
tier3 = baseline admin network policy